swap_meet/clothing.py

```diff
@@ -1,2 +1,13 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+
+class Clothing(Item):
+    
+    category = "Clothing"
+    
+    def __init__(self, condition=0, age=None):
+        super().__init__(type(self).category, condition, age)
```

Comment:
This is really cool!

Did you try this without specifying the `__init__` in the child classes?

It looks pretty close to what you'd need.

Reply:
Yes I tried that method we discussed during the review session, but the class attribute `category` gets overridden by the instance attribute `self.category` when the instance is initialized. So some of the tests didnt pass.

Reply:
Ah, yeah.  

I think you want the `Item`s constructor to just initialize `self.category` to `type(self).__name__`.  It's a little bit of a hack anyway though.